packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -74,16 +88,38 @@ def imshow(img, zmin=None, zmax=None, origin=None, colorscale=None):
         zmin and zmax correspond to the min and max values of the datatype for integer
         datatypes (ie [0-255] for uint8 images, [0, 65535] for uint16 images, etc.). For
         a multichannel image of floats, the max of the image is computed and zmax is the
-        smallest power of 256 (1, 255, 65535) greater than this max value, 
+        smallest power of 256 (1, 255, 65535) greater than this max value,
         with a 5% tolerance. For a single-channel image, the max of the image is used.
 
     origin : str, 'upper' or 'lower' (default 'upper')
         position of the [0, 0] pixel of the image array, in the upper left or lower left
         corner. The convention 'upper' is typically used for matrices and images.
 
-    colorscale : str
-        colormap used to map scalar data to colors (for a 2D image). This parameter is not used for
-        RGB or RGBA images.
+    color_continuous_scale : str or list of str
```

Comment:
`str (name of colorscale) or list of str (list of css colors)` would be clearer here.

Reply:
ok it's explained below, it's fine